Makefile

```diff
@@ -0,0 +1,17 @@
+setup:
+	@rustup component add rustfmt
+	@rustup component add clippy
+	@type -p pre-commit >/dev/null 2>&1 || \
+		(echo "Please install pre-commit and try again"; exit 1)
+	@pre-commit install -f --hook-type pre-commit
+	@pre-commit install -f --hook-type pre-push
+
+mongo:
+	@docker run --name wallet-api-mongo --rm -p 27017:27017 \
```

Comment:
Where is the Dockerfile? I think we need one, right?

Reply:
nah! my bad! `-d mongo` =)

Reply:
Good that you mention that line, as I totally forgot to comment that this command fails when there's no `datadir` directory:

```
docker: Error response from daemon: invalid mount config for type "bind": bind source path does not exist: /.../wallet-api/datadir.
```

So, what would you guys prefer?

1. Having a `datadir` with a `.gitignore` inside — and nothing else in case you run mongo locally
2. Manually creating it after `make mongo` fails with that message

Reply:
/nudge @marcelometal @kov

Reply:
I'd prefer `mkdir -p datadir` at the beginning of the rule

Reply:
Yeah, I thought of that too. Dunno why I didn't mention it. But, the PR is already merged 😆

Reply:
Metal pé quente

Reply:
hahhaha malz aê nem vi essa thread =/ so vi que geral tinha aprovado.... na proxima bora mudar o status

Reply:
Aí complica. Vou mandar na `master` direto..